server/index.ts

```diff
@@ -0,0 +1,126 @@
+import Koa from 'koa'
+import Router from 'koa-router'
+import next from 'next'
+import createShopifyAuth, { verifyRequest } from "@shopify/koa-shopify-auth";
+import Shopify, { ApiVersion } from "@shopify/shopify-api";
+import dotenv from 'dotenv';
+
+dotenv.config()
+
+const {
+  PORT,
+  NODE_ENV,
+  SHOPIFY_API_KEY,
+  SHOPIFY_API_SECRET,
+  SCOPES,
+  HOST
+} = process.env as any
```

Comment:
Instead of directly using `process.env`, can we try to look into a dedicated configuration package? Would recommend this - https://www.npmjs.com/package/config

The idea behind using this package is:
- You can easily create new environments (staging / production / testing / etc)
- As you have a centralised place for obtaining the config, it becomes easier to mock it during testing.